incognia.go

```diff
@@ -7,11 +7,16 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"runtime"
+	"syscall"
 	"time"
 )
 
 const (
 	defaultNetClientTimeout = 5 * time.Second
+	contentTypeKey          = "Content-Type"
+	userAgentKey            = "User-Agent"
+	libVersion              = "1.8.0"
```

Comment:
This shouldn't be hardcoded and must be injected in compile time. Something like:

```bash
go build -ldflags="-X 'repo.incognia.com/go/incognia.libVersion=$(git describe --always --dirty --long)'"
```

Example:
https://github.com/inloco/entrypoint-tool/blob/615aee22aabb6f7126f6224745296c6c6a95326d/Makefile#L26-L36
https://github.com/inloco/entrypoint-tool/blob/615aee22aabb6f7126f6224745296c6c6a95326d/version/main.go#L8

Reply:
Okay, i'm on it